.github/CODEOWNERS

```diff
@@ -0,0 +1,63 @@
+.github/CODEOWNERS @alpenlabs/admins             # only admins should own the CODEOWNERS
+.gitignore         @alpenlabs/admins
+.github/*TEMPLATE  @alpenlabs/admins             # Cover PRs and Issues
+**/Cargo.toml      @alpenlabs/dependencies       # dependencies and metadata
+**/Cargo.lock      @alpenlabs/dependencies       # updates
+.config/           @alpenlabs/rust @alpenlabs/ci # config for rustdev and CI
+**/*.toml          @alpenlabs/rust               # config files
+
+# CI/Local ENV
+.github/workflows/     @alpenlabs/ci
+.github/actions/       @alpenlabs/ci
+.github/dependabot.yml @alpenlabs/ci
```

Comment:
Is it possible to glob all of `.github/` to the admin group, and then special-case these  particular files/subdirectories to the CI group? Seems like it would be safer overall, since other folks probably shouldn't be messing around in there.

Reply:
It will trigger both in case of a `.github/workflows/foo.yml` PR change

Reply:
Rats. So you can't get as fine-grained as we might like for that?

Reply:
The docs says that it follows `.gitignore` conventions however some things are not supported like `!` negations and ranges with `[ ]`.

Ref: https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#codeowners-syntax

Reply:
From [this example](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#example-of-a-codeowners-file), it looks like only the last matching pattern takes precedence.

Reply:
Oh great! I'll refactor it

Reply:
I think this means things like the CI globs near the top would need to be moved down so they aren't overwritten.

Reply:
Check now